kunta-api-server/src/test/java/fi/otavanopisto/kuntaapi/test/server/integrations/mikkelinyt/MikkeliNytRestTestsIT.java

```diff
@@ -0,0 +1,348 @@
+      event.setPlace(formatPlace(place));
+      event.setStart(formatDateTime(start));
+      event.setThumb(thumb);
+      event.setUrl(url);
+      event.setZip(formatZip(zip));
+      
+      Attachment imageAttachment = new Attachment();
+      imageAttachment.setContentType(ATTACHMENT_TYPE);
+      imageAttachment.setSize(new Long(ATTACHMENT_SIZE));
+      imageAttachment.setId(kuntaApiAttachmentId);
+      
+      this.events.add(event);
+      
+      return event;
+    }
+    
+    private String formatAddress(String address) {
+      return String.format("<span class=\"Address\">%s</span>", address);
+    }
+    
+    private String formatZip(String zip) {
+      return String.format("<span class=\"Zip\">%s</span>", zip);
+    }
+    
+    private String formatCity(String zip) {
+      return String.format("<span class=\"City\">%s</span>", zip);
+    }
+    
+    private String formatPlace(String place) {
+      return String.format("<span class=\"Place\">%s</span>", place);
+    }
+    
+    private String formatDateTime(OffsetDateTime localDateTime) {
+      DateTimeFormatter formatter = new DateTimeFormatterBuilder()
+        .parseCaseInsensitive()
+        .append(DateTimeFormatter.ISO_LOCAL_DATE)
+        .appendLiteral(' ')
+        .append(DateTimeFormatter.ISO_LOCAL_TIME)
+        .toFormatter();
+      
+      return localDateTime.format(formatter);
+    }
+    
+    private String formatNiceDateRange(OffsetDateTime from, OffsetDateTime to) {
+      DateTimeFormatter startDateFormat = new DateTimeFormatterBuilder()
+        .appendValue(ChronoField.DAY_OF_MONTH)
+        .appendLiteral('.')
+        .appendValue(ChronoField.MONTH_OF_YEAR)
+        .toFormatter();
+      
+      DateTimeFormatter endDateFormat = new DateTimeFormatterBuilder()
+          .appendValue(ChronoField.DAY_OF_MONTH)
+          .appendLiteral('.')
+          .appendValue(ChronoField.MONTH_OF_YEAR)
+          .appendLiteral('.')
+          .appendValue(ChronoField.YEAR)
+          .toFormatter();
+      
+      DateTimeFormatter timeFormat = new DateTimeFormatterBuilder()
+          .appendValue(ChronoField.HOUR_OF_DAY)
+          .appendLiteral(':')
+          .appendValue(ChronoField.MINUTE_OF_HOUR)
+          .toFormatter();
+      
+      return String.format("%s - %s klo %s - %s", 
+          from.format(startDateFormat), 
+          to.format(endDateFormat), 
+          from.format(timeFormat), 
+          to.format(timeFormat));
+    }
+    
+    @Override
+    public void startMock() {
+      EventsResponse eventsResponse = new EventsResponse();
+      eventsResponse.setData(events);
+      eventsResponse.setMessage(String.format("Löytyi %d tapahtumaa", events.size()));
+      eventsResponse.setStatus(1);
+      mockGetJSON(String.format("/json.php?apiKey=%s&location=%s", ORGANIZATION_SETTING_APIKEY, ORGANIZATION_SETTING_LOCATION), eventsResponse);
+      super.startMock();
+    }
+    
+    private List<Event> events;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)